<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>761</width>
    <height>288</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>Mini DMX</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <layout class="QGridLayout" name="gridLayout">
      <item row="0" column="0">
       <layout class="QHBoxLayout" name="horizontalLayout_2">
        <item>
         <widget class="QGroupBox" name="groupBox">
          <property name="title">
           <string>Int</string>
          </property>
          <widget class="QSlider" name="verticalSlider">
           <property name="geometry">
            <rect>
             <x>20</x>
             <y>20</y>
             <width>19</width>
             <height>160</height>
            </rect>
           </property>
           <property name="maximum">
            <number>255</number>
           </property>
           <property name="singleStep">
            <number>1</number>
           </property>
           <property name="orientation">
            <enum>Qt::Vertical</enum>
           </property>
          </widget>
          <widget class="QSpinBox" name="levelSpinBox">
           <property name="geometry">
            <rect>
             <x>10</x>
             <y>200</y>
             <width>42</width>
             <height>22</height>
            </rect>
           </property>
           <property name="maximum">
            <number>255</number>
           </property>
          </widget>
         </widget>
        </item>
        <item>
         <widget class="QGroupBox" name="groupBox_2">
          <property name="title">
           <string>Red</string>
          </property>
          <widget class="QSlider" name="verticalSlider_2">
           <property name="geometry">
            <rect>
             <x>20</x>
             <y>20</y>
             <width>19</width>
             <height>160</height>
            </rect>
           </property>
           <property name="maximum">
            <number>255</number>
           </property>
           <property name="singleStep">
            <number>1</number>
           </property>
           <property name="orientation">
            <enum>Qt::Vertical</enum>
           </property>
          </widget>
          <widget class="QSpinBox" name="rSpinBox">
           <property name="geometry">
            <rect>
             <x>10</x>
             <y>200</y>
             <width>42</width>
             <height>22</height>
            </rect>
           </property>
           <property name="maximum">
            <number>255</number>
           </property>
          </widget>
         </widget>
        </item>
        <item>
         <widget class="QGroupBox" name="groupBox_3">
          <property name="title">
           <string>Green</string>
          </property>
          <widget class="QSlider" name="verticalSlider_3">
           <property name="geometry">
            <rect>
             <x>20</x>
             <y>20</y>
             <width>19</width>
             <height>160</height>
            </rect>
           </property>
           <property name="maximum">
            <number>255</number>
           </property>
           <property name="singleStep">
            <number>1</number>
           </property>
           <property name="orientation">
            <enum>Qt::Vertical</enum>
           </property>
          </widget>
          <widget class="QSpinBox" name="gSpinBox">
           <property name="geometry">
            <rect>
             <x>10</x>
             <y>200</y>
             <width>42</width>
             <height>22</height>
            </rect>
           </property>
           <property name="maximum">
            <number>255</number>
           </property>
          </widget>
         </widget>
        </item>
        <item>
         <widget class="QGroupBox" name="groupBox_4">
          <property name="title">
           <string>Blue</string>
          </property>
          <widget class="QSlider" name="verticalSlider_4">
           <property name="geometry">
            <rect>
             <x>20</x>
             <y>20</y>
             <width>19</width>
             <height>160</height>
            </rect>
           </property>
           <property name="maximum">
            <number>255</number>
           </property>
           <property name="singleStep">
            <number>1</number>
           </property>
           <property name="orientation">
            <enum>Qt::Vertical</enum>
           </property>
          </widget>
          <widget class="QSpinBox" name="bSpinBox">
           <property name="geometry">
            <rect>
             <x>10</x>
             <y>200</y>
             <width>42</width>
             <height>22</height>
            </rect>
           </property>
           <property name="maximum">
            <number>255</number>
           </property>
          </widget>
         </widget>
        </item>
        <item>
         <widget class="QGroupBox" name="groupBox_5">
          <property name="title">
           <string>Strobe</string>
          </property>
          <widget class="QSlider" name="verticalSlider_5">
           <property name="geometry">
            <rect>
             <x>20</x>
             <y>20</y>
             <width>19</width>
             <height>160</height>
            </rect>
           </property>
           <property name="maximum">
            <number>255</number>
           </property>
           <property name="singleStep">
            <number>1</number>
           </property>
           <property name="orientation">
            <enum>Qt::Vertical</enum>
           </property>
          </widget>
          <widget class="QSpinBox" name="strobeSpinBox">
           <property name="geometry">
            <rect>
             <x>10</x>
             <y>200</y>
             <width>42</width>
             <height>22</height>
            </rect>
           </property>
           <property name="maximum">
            <number>255</number>
           </property>
          </widget>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>761</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections>
  <connection>
   <sender>verticalSlider</sender>
   <signal>sliderMoved(int)</signal>
   <receiver>levelSpinBox</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>39</x>
     <y>142</y>
    </hint>
    <hint type="destinationlabel">
     <x>40</x>
     <y>253</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>levelSpinBox</sender>
   <signal>valueChanged(int)</signal>
   <receiver>verticalSlider</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>40</x>
     <y>253</y>
    </hint>
    <hint type="destinationlabel">
     <x>39</x>
     <y>142</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>levelSpinBox</sender>
   <signal>valueChanged(int)</signal>
   <receiver>MainWindow</receiver>
   <slot>faderSliderMoved()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>40</x>
     <y>253</y>
    </hint>
    <hint type="destinationlabel">
     <x>199</x>
     <y>149</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>verticalSlider</sender>
   <signal>sliderMoved(int)</signal>
   <receiver>MainWindow</receiver>
   <slot>faderSliderMoved()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>39</x>
     <y>142</y>
    </hint>
    <hint type="destinationlabel">
     <x>199</x>
     <y>149</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>rSpinBox</sender>
   <signal>valueChanged(int)</signal>
   <receiver>verticalSlider_2</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>111</x>
     <y>252</y>
    </hint>
    <hint type="destinationlabel">
     <x>110</x>
     <y>186</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>verticalSlider_2</sender>
   <signal>sliderMoved(int)</signal>
   <receiver>rSpinBox</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>111</x>
     <y>119</y>
    </hint>
    <hint type="destinationlabel">
     <x>122</x>
     <y>257</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>gSpinBox</sender>
   <signal>valueChanged(int)</signal>
   <receiver>verticalSlider_3</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>177</x>
     <y>256</y>
    </hint>
    <hint type="destinationlabel">
     <x>180</x>
     <y>202</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>bSpinBox</sender>
   <signal>valueChanged(int)</signal>
   <receiver>verticalSlider_4</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>259</x>
     <y>249</y>
    </hint>
    <hint type="destinationlabel">
     <x>263</x>
     <y>217</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>verticalSlider_4</sender>
   <signal>sliderMoved(int)</signal>
   <receiver>bSpinBox</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>265</x>
     <y>113</y>
    </hint>
    <hint type="destinationlabel">
     <x>258</x>
     <y>259</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>verticalSlider_3</sender>
   <signal>sliderMoved(int)</signal>
   <receiver>gSpinBox</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>181</x>
     <y>99</y>
    </hint>
    <hint type="destinationlabel">
     <x>194</x>
     <y>253</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>rSpinBox</sender>
   <signal>valueChanged(int)</signal>
   <receiver>MainWindow</receiver>
   <slot>faderSliderMoved()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>101</x>
     <y>258</y>
    </hint>
    <hint type="destinationlabel">
     <x>144</x>
     <y>251</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>gSpinBox</sender>
   <signal>valueChanged(int)</signal>
   <receiver>MainWindow</receiver>
   <slot>faderSliderMoved()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>178</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>214</x>
     <y>267</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>bSpinBox</sender>
   <signal>valueChanged(int)</signal>
   <receiver>MainWindow</receiver>
   <slot>faderSliderMoved()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>269</x>
     <y>250</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>247</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>verticalSlider_4</sender>
   <signal>sliderMoved(int)</signal>
   <receiver>MainWindow</receiver>
   <slot>faderSliderMoved()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>270</x>
     <y>117</y>
    </hint>
    <hint type="destinationlabel">
     <x>214</x>
     <y>106</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>verticalSlider_3</sender>
   <signal>sliderMoved(int)</signal>
   <receiver>MainWindow</receiver>
   <slot>faderSliderMoved()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>183</x>
     <y>130</y>
    </hint>
    <hint type="destinationlabel">
     <x>145</x>
     <y>132</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>verticalSlider_2</sender>
   <signal>sliderMoved(int)</signal>
   <receiver>MainWindow</receiver>
   <slot>faderSliderMoved()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>108</x>
     <y>80</y>
    </hint>
    <hint type="destinationlabel">
     <x>76</x>
     <y>89</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>verticalSlider_5</sender>
   <signal>sliderMoved(int)</signal>
   <receiver>strobeSpinBox</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>342</x>
     <y>184</y>
    </hint>
    <hint type="destinationlabel">
     <x>342</x>
     <y>258</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>strobeSpinBox</sender>
   <signal>valueChanged(int)</signal>
   <receiver>verticalSlider_5</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>356</x>
     <y>258</y>
    </hint>
    <hint type="destinationlabel">
     <x>340</x>
     <y>116</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>verticalSlider_5</sender>
   <signal>sliderMoved(int)</signal>
   <receiver>MainWindow</receiver>
   <slot>faderSliderMoved()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>344</x>
     <y>120</y>
    </hint>
    <hint type="destinationlabel">
     <x>308</x>
     <y>153</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>strobeSpinBox</sender>
   <signal>valueChanged(int)</signal>
   <receiver>MainWindow</receiver>
   <slot>faderSliderMoved()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>330</x>
     <y>251</y>
    </hint>
    <hint type="destinationlabel">
     <x>309</x>
     <y>213</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>faderSliderMoved()</slot>
 </slots>
</ui>
